ethcore/private-tx/src/encryptor.rs

```diff
@@ -86,12 +86,11 @@ pub struct SecretStoreEncryptor {
 
 impl SecretStoreEncryptor {
 	/// Create new encryptor
-	pub fn new(config: EncryptorConfig) -> Result<Self, Error> {
+	pub fn new(config: EncryptorConfig, client: FetchClient) -> Result<Self, Error> {
 		Ok(SecretStoreEncryptor {
-			config: config,
-			client: FetchClient::new()
-				.map_err(|e| ErrorKind::Encrypt(format!("{}", e)))?,
```

Comment:
We should reuse existing `FetchClient`, cause each new instance of it creates new `CpuPool`.